libafl_cc/src/lib.rs

```diff
@@ -168,7 +168,7 @@ pub trait CompilerWrapper {
         let args = self.command()?;
 
         if !self.is_silent() {
-            log::info!("{args:?}");
```

Comment:
Why? We also want log in libafl_cc imho - then users can change the verbosity as they wish